moveit_kinematics/ikfast_kinematics_plugin/templates/ikfast61_moveit_plugin_template.cpp

```diff
@@ -409,18 +412,30 @@ bool IKFastKinematicsPlugin::initialize(const moveit::core::RobotModel& robot_mo
   }
 
   storeValues(robot_model, group_name, base_frame, tip_frames, search_discretization);
+  if (!lookupParam("linkprefix", linkframeprefix_, std::string("")))
+  {
+    ROS_INFO_NAMED(name_, "linkprefix parameter not set. Assuming empty prefix.");
```

Comment:
This should be a ROS_WARN

Reply:
I would not expect *any* output here, although an info could be justified.
It's perfectly valid to run the plugin without a prefix. Why change that?

Reply:
I do understand your motivation to have a warning, but as a consequence this would add new warnings to all setups that use the plugins (and regenerate them).
If you think the additional warning (before the fatal error later on you will surely get when it tries to access nonexisting joints) is worth this breakage, I do not mind too much.
Personally, I would prefer to reduce output for current users over changing interfaces.

Reply:
For single-arm-setups, it's valid to leave the prefix empty. Thus it shouldn't be a warning. I'm increasing verbosity in the "computeRelativeTransform" part to give good Errors, if the expected frame is missing.